features/attached_commands.feature

```diff
@@ -434,7 +434,8 @@ Feature: Command behaviour when attached to an UA subscription
         update_status_timer     +43200
         gcp_auto_attach_timer   +0
         """
-        And I verify that running `grep "Disabling gcp_auto_attach job" /var/log/ubuntu-advantage.log` `with sudo` exits `0`
+        And I verify that running `grep "Disabling gcp_auto_attach job" /var/log/ubuntu-advantage-timer.log` `with sudo` exits `0`
```

Comment:
When running the test locally, this entry does not appear on the `/var/log/ubuntu-advantage-timer.log` log or `/var/log/ubuntu-advantage.log`

Reply:
This is my bad - fixed now.
I had a shared logger for the timer and job, and then dismissed it because it didn't look so right.
Added a setup_logging call to the gcp job matching the file to ensure it.

Weirdly, my local tests were passing :(